src/cli/index.ts

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+import { processArgs } from "./cli";
+processArgs()();
```

Comment:
This feels weird to me, why is `processArgs` a function that returns a function if both are void and used without currying?
Maybe `processArgs` can call the pipe defined as `logToConsole` and simply return void instead?

Reply:
It just ensures that `processArgs` is pure and the side-effect is applied only at the very extreme of the CLI. That enables, for example, writing tests for `processArgs` without printing to console (or file if we add that option).